index.ts

```diff
@@ -91,6 +91,26 @@ export class JolocomSDK {
     }
   }
 
+  /**
+   * Creates a signed, base64 encoded Authentication Request, given a
+   * callbackURL
+   *
+   * @param callbackURL - the callbackURL to which the Authentication Response
+   *                      should be sent
+   * @returns Base64 encoded signed Authentication Request
+   */
+  public async authRequestToken(callbackURL: string): Promise<string> {
```

Comment:
This whole interface of creating request tokens here needs to be lifted down into InteractionManager/Interaction/Flow
At the SDK level we should probably export an API for `startAuthInteraction` or some such higher level than tokens